src/app.js

```diff
@@ -23,7 +24,7 @@ let model;
 handTrack.startVideo(video).then(status => {
     if(status){
         navigator.getUserMedia(
-            {video: {}}, 
+            {video: {}},
```

Comment:
This part can be deleted, the main branch has the code needed to run handtrack.js